src/core/lib/security/security_connector/load_system_roots.h

```diff
@@ -19,16 +19,14 @@
 #ifndef GRPC_SRC_CORE_LIB_SECURITY_SECURITY_CONNECTOR_LOAD_SYSTEM_ROOTS_H
 #define GRPC_SRC_CORE_LIB_SECURITY_SECURITY_CONNECTOR_LOAD_SYSTEM_ROOTS_H
 
-#include <grpc/slice.h>
 #include <grpc/support/port_platform.h>
 
-namespace grpc_core {
+#include "src/core/lib/slice/slice.h"
 
-// TODO(matthewstevenson88): Update LoadSystemRootCerts to use Slice
-// instead of grpc_slice.
+namespace grpc_core {
 
 // Returns a slice containing roots from the OS trust store
-grpc_slice LoadSystemRootCerts();
+Slice LoadSystemRootCerts();
```

Comment:
While we're at it, how about changing this to return `absl::StatusOr<Slice>`, so that we have a more obvious way of returning an error?